src/cartesi-machine.lua

```diff
@@ -1358,20 +1453,61 @@ else
         uarch = uarch,
         flash_drive = {},
     }
+
+    -- show splash on init
+    if init_splash then
+        config.dtb.init = config.dtb.init
+            .. ([[
+echo "
+         .
+        / \
+      /    \
+\---/---\  /----\
+ \       X       \
+  \----/  \---/---\
+       \    / CARTESI
+        \ /   MACHINE
+         '
+"
+]]):gsub("\\", "\\\\")
+    end
+
     for _, label in ipairs(flash_label_order) do
+        local devname = "pmem" .. #config.flash_drive
         config.flash_drive[#config.flash_drive + 1] = {
             image_filename = flash_image_filename[label],
             shared = flash_shared[label],
             start = flash_start[label],
             length = flash_length[label] or -1,
         }
+        -- auto mount
+        local mount = flash_mount[label]
+        local chownpath = "/dev/" .. devname
+        if label ~= "root" and mount then
+            local cmd = ([[busybox mkdir -p "MOUNT" && busybox mount /dev/DEVNAME "MOUNT"]])
+                :gsub("MOUNT", mount)
```

Comment:
Can the user pick MOUT=DEVNAME? :smile:

Reply:
Good catch, I fixed the issue in case user picks that name :) , howeever I didn't went further with a concat across multiple lines.